draft-ietf-quic-http.md

```diff
@@ -777,18 +777,13 @@ Settings which are integers use the QUIC variable-length integer encoding.
 
 #### Defined SETTINGS Parameters {#settings-parameters}
 
-The following settings are defined in HTTP/QUIC:
-
-  SETTINGS_HEADER_TABLE_SIZE (0x1):
-  : An integer with a maximum value of 2^30 - 1.  The default value is 4,096
-    bytes.
+The following setting is defined in HTTP/QUIC:
 
   SETTINGS_MAX_HEADER_LIST_SIZE (0x6):
   : An integer with a maximum value of 2^30 - 1.  The default value is
     unlimited.
 
-  SETTINGS_QPACK_BLOCKED_STREAMS (0x7):
-  : An integer with a maximum value of 2^16 - 1.  The default value is 100.
+Additional settings MAY be defined by extensions to HTTP/QUIC.
```

Comment:
Are we calling QPACK an extension to HTTP/QUIC?  Or it is for purposes of SETTINGS?

Reply:
Kind of.  It's required to parse HEADERS and PUSH_PROMISE, but it is its own thing.  Kind of a mandatory extension, I suppose?